themes/mongodb/src/widgets/widgets.js

```diff
@@ -0,0 +1,75 @@
+import Constants from './Constants';
+import Deluge from './deluge/deluge';
+import PropTypes from 'prop-types';
+import Suggestion from './suggestion/Suggestion';
+import preact from 'preact';
+
+class Widgets extends preact.Component {
+    constructor(props) {
+        super(props);
+        this.isSuggestionPage = this.isSuggestionPage(this.props.path);
+        this.handleOpenDrawer = this.handleOpenDrawer.bind(this);
+        this.handleCloseDrawer = this.handleCloseDrawer.bind(this);
+
+        this.state = {
+            'drawerIsOpen': false,
+            'drawerHasOpened': false
+        };
+    }
+
+    isSuggestionPage(path) {
+        return Constants.SUGGESTION_WHITELIST.includes(path);
```

Comment:
I, uh, might have been unclear :D

This is okay, but I actually just meant it's better to use a global variable within this module rather than an unchanging instance variable

Reply:
Oh whoops. So you just suggest moving it outside of the `Widgets` class, right?

Reply:
Yup!